installers/include/wrapper-properties.go-server.conf.example

```diff
@@ -34,4 +35,5 @@
 #wrapper.java.additional.104.stripquotes=TRUE
 
 # Set a memory limit of 8GB (usually needed on your GoCD server for large setups). We recommend that you do not use more than half your system memory.
-#wrapper.java.additional.105=-Xmx8G
+# This needs to be index 2, since it needs to override the Xmx set in wrapper.conf
+#wrapper.java.additional.2=-Xmx8G
```

Comment:
It is my understanding is that `java -Xmx1g -Xmx2g` will force the JVM to use the heap specified last. The intention of using indices > 100 was that it allows GoCD to reserve indices upto 100 for its internal use.

Reply:
Ah, sorry. I didn't know that a second one overrides the previous one. Then, it makes sense to leave it at 105. I force-pushed over it to remove that change.